go.mod

```diff
@@ -69,6 +69,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.13.0
 )
 
+require github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v0.5.1 // indirect
```

Comment:
Why did this end up in its own block?

Reply:
Not sure, running `make modtidy-all` added it in this way. I have placed it in the correct block now.